UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -129,7 +129,7 @@ public enum RoomEntryFlags : uint
     /// <summary>
     /// The thickness of the room grid in pixels.
     /// </summary>
-    public double GridThicknessPx { get; set; } = 1d;
+    public double GridThicknessPx { get; set; } = 0.5d;
```

Comment:
Obviously preference, but I think reverting the grid thickness would be better. Currently it looks way too thin IMO.